.eslintrc.json

```diff
@@ -42,7 +42,9 @@
     "@typescript-eslint/no-unnecessary-type-assertion": "warn",
     "@typescript-eslint/no-non-null-assertion": "off",
     "@typescript-eslint/no-non-null-asserted-optional-chain": "off",
-    "@typescript-eslint/explicit-module-boundary-types": "off"
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-extra-semi": "warn"
```

Comment:
- We use empty no-op functions all over the place, the rule isn't helpful
- Prettier may enforce adding extra semicolons where TSLint doesn't like them, so changing that to a warning